core/src/test/java/greencity/mapping/AddEcoNewsDtoRequestMapperTest.java

```diff
@@ -29,33 +36,48 @@
     private LanguageRepository languageRepository;
 
     private Language language = new Language(1L, "en", Collections.emptyList(), Collections.emptyList(),
-        Collections.emptyList());
+            Collections.emptyList());
+
+    private EcoNewsTranslation ecoNewsTranslation = new EcoNewsTranslation(1L, language, "title", "text", null);
 
-    private EcoNewsTranslation ecoNewsTranslation = new EcoNewsTranslation(1L, language, "title", null);
+    private User author =
```

Comment:
Better to have some test util class, for example ModelUtils, where you can prepare needed for tests models to not add this code to tests

Reply:
It will also helps you avoid duplication in other tests